### ISSUE TITLE
Big Button focus state accessibility issues / text hidden

### ISSUE THREAD

**Issue by trubach** [ATLAS-Illinois/Illinois-Framework-Theme#1022]
**Describe the bug**
When tabbing through the big buttons:

1) There is no visual indication of focus state on buttons with a blue background
2) On white background buttons, the focus state sets the background to blue, which causes the text to be hidden (the text color is blue)

**To Reproduce**
1. Go to https://drupal.webtheme.illinois.edu/paragraphs/big-buttons
2. Tab through the big buttons

**Expected behavior**
The Big Button's focus state should be distinct for the blue background buttons and the text should be white on the focus state of the white background buttons (when it changes to a blue background). At a minimum, the focus state should match the hover state.

**Screenshots**

![Image](https://github.com/user-attachments/assets/e3afb5de-75cb-43af-8892-1d5167c92b1b)

**Additional context**
This was reported by Robert Slater while working on the new Digital Accessibility website
